### ISSUE TITLE
Struggling with using 2dgs. --use_2dgs

### ISSUE THREAD

**Issue by Jaewon040** [vye16/shape-of-motion#82]
Hi! Thank you for your great work. It gives us a lot of intuitions.
I want to ask you (or just someone who read this issue) is there any problem when training with 2dgs?
Because in my case, i kept failed to use 2dgs.

After 3~10 global step, 
./flow3d/trainer.py(484)compute_losses() stops me.
if self.model.fg.params["means"].isnan().sum() > 0:
     import ipdb
     ipdb.set_trace()

loss became NaN and

ipdb>   for key, param in self.model.fg.params.items():
      nan_count = param.isnan().sum()
      if nan_count > 0:
          print(f"{key} has {nan_count} NaN values")
 
>  means has 82506 NaN values
    motion_coefs has 275020 NaN values
    opacities has 27502 NaN values
    quats has 110008 NaN values
    scales has 55004 NaN values

is there any way to solve this problem?
I am using Gsplat 1.4.0 (In my case, 1.5.x doesn't work in 2dgs)
python -c "import torch; print(torch.__version__)"
2.2.0+cu121

**Comment by MagicTZ** [comment]
Did you solve it? I got the same issue

**Comment by Jaewon040** [comment]
@MagicTZ Unfortunately not yet ,,
